_assets/js/netlify/topics/makeTemplate.js

```diff
@@ -0,0 +1,46 @@
+import generateChildElements from './generateChildElements';
```

Comment:
not a blocker, but want to note somewhere that we should look into whether JSX is usable